core/src/main/kotlin/org/evomaster/core/output/service/TestSuiteWriter.kt

```diff
@@ -68,13 +71,34 @@ class TestSuiteWriter {
 
             beforeAfterMethods(controllerName, lines)
 
-            val tests = testSuiteOrganizer.sortTests(solution, config.customNaming)
+            //catch any sorting problems (see NPE is SortingHelper on Trello)
+            val tests = try{
+                testSuiteOrganizer.sortTests(solution, config.customNaming)
+            }
+            catch (ex: Exception){
+                var counter = 0
+                log.warn("A failure has occurred with the test sorting. Reverting to default settings. \n"
+                        + "Exception: ${ex.localizedMessage} \n"
+                        + "At ${ex.stackTrace.joinToString(separator = " \n -> ")}. ")
+                solution.individuals.map { ind -> TestCase(ind, "test_${counter++}") }
+            }
 
             for (test in tests) {
                 lines.addEmpty(2)
 
-                val testLines = TestCaseWriter()
-                        .convertToCompilableTestCode(config, test, baseUrlOfSut)
+                // catch writing problems on an individual test case basis
+                val testLines = try {
+                    TestCaseWriter()
+                            .convertToCompilableTestCode(config, test, baseUrlOfSut)
+
+                }
+                catch (ex: Exception){
+                    Lines().apply {
+                        log.warn("A failure has occurred in writing test ${test.name}. \n "
```

Comment:
what is the reason of using `Lines().apply` here?

Reply:
Mostly leftover from previous tries (where I was writing that stuff to the test file).
I am modifying it now to return an empty Lines() object (since that is what val testLines expects for each individual test case).